junit5/src/main/java/cz/xtf/junit5/listeners/ProjectCreator.java

```diff
@@ -20,6 +20,12 @@ public void testPlanExecutionStarted(TestPlan testPlan) {
         if (openShift.getProject() == null) {
             openShift.createProjectRequest();
             openShift.waiters().isProjectReady().waitFor();
+            // Adding a label can be only done via 'namespace'. It cannot be set via 'project' API.
+            // Thus we do this separately. Also, updating namespaces seems to be restricted only to Admin users.
+            OpenShifts.admin().namespaces().withName(
```

Comment:
Whenever using `.admin()` please specify in comment which permission we care about. This would be helpful for environments not having admin permissions.

In this case `patch resource "namespaces" in API group "" in the namespace "???"`

Reply:
I've updated in the comment what you mentioned.